### PR TITLE
FISH-707 add 5.2020.6 Jakarta TCK results

### PR DESCRIPTION
**Pull request by MarkWareham**
as title

### CODE REVIEW

File: docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-full-5.2020.6.adoc [hunk 1]
```diff
@@ -0,0 +1,753 @@
...
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] Results:
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] Tests run: 1805, Failures: 0, Errors: 0, Skipped: 0
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] --- maven-surefire-report-plugin:2.19.1:report-only (generate-test-report) @ weld-payara-runner-tck11 ---
+ [mvn.test] [WARNING] Unable to locate Test Source XRef to link to - DISABLED
+ [mvn.test] [INFO] ------------------------------------------------------------------------
+ [mvn.test] [INFO] BUILD SUCCESS
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-full-5.2020.6.adoc:102]: Trim this bit out - 102 to line 97 can be removed

File: docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-full-5.2020.6.adoc [hunk 2]
```diff
@@ -0,0 +1,753 @@
...
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 1043, Failures: 0, Errors: 0, Skipped: 0
+[INFO] 
+[INFO] 
+[INFO] --- maven-surefire-report-plugin:2.21.0:report-only (generate-test-report) @ hibernate-validator-tck-runner ---
+[WARNING] Unable to locate Test Source XRef to link to - DISABLED
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-full-5.2020.6.adoc:87]: Trim this bit out - 87 to line 82 can be removed

**Review by AlanRoth — CHANGES_REQUESTED** [review]

**Review by rdebusscher — APPROVED** [review]
Based on the 5.2020.5 which is missing the Jakarta Compatible logo but that I'll fix that after this PR is merged.

File: docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-full-5.2020.6.adoc (hunk 1 above)
Comment (MarkWareham) [reply docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-full-5.2020.6.adoc:102]: hmm, didn't spot that. Seems porting packages output differently

**Review by AlanRoth — APPROVED** [review]

**Pull request merged**